pcp/PCPDynamicScreen.c

```diff
@@ -0,0 +1,413 @@
+/*
+htop - PCPDynamicScreen.c
+(C) 2022 Sohaib Mohammed
+(C) 2022-2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "pcp/PCPDynamicScreen.h"
+
+#include <ctype.h>
+#include <dirent.h>
+#include <pcp/pmapi.h>
```

Comment:
Order? Or specific reason for this order? Comment if so.

Reply:
@BenBE not sure what you mean here - the order looks correct to me (alphabetical).

Reply:
I think when I marked this, this block also contained some headers out of sort order. But this still applies for the block directly below. Just re-check for the pcp/ files for the header order.

Reply:
Ah yes, I see it now - thanks.